src/client/containers/Channel/Channel.js

```diff
@@ -0,0 +1,35 @@
+import React, { useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
+import { UserContext } from '../../context/userContext';
+
+export default function Channel() {
+  const { id } = useParams();
+  const [messages, setMessages] = useState([]);
+
+  useEffect(() => {
+    const fetchMessages = async () => {
+      try {
+        const url = `http://localhost:3000/api/messages?channel_id=${id}`;
```

Comment:
```suggestion
        const url = `/api/messages?channel_id=${id}`;
```

This wont work in production since the site wont be localhost

Reply:
You're right. I'll changed to use the base url dynamically.

Reply:
Fixed